src/views/Project/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from 'material-ui/Typography';
+import ReactMarkdown from 'react-markdown';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fileName: props.match.params.projectKey,
+      projects,
+    };
+  }
+  render() {
+    const { fileName, projects } = this.state;
+    const projectInfo = projects[fileName];
+
+    return (
+      <div>
+        <header>
+          <Grid
+            container
+            spacing={16}
+            direction="column"
+            justify="center"
+            alignItems="center">
+            <Grid item xs={12}>
+              <div>
+                <div />
+                <div>
+                  <Typography variant="display4" align="center">
+                    {projectInfo.name}
+                  </Typography>
+                  {projectInfo.description && (
+                    <Typography align="center">
+                      <ReactMarkdown source={projectInfo.description} />
+                    </Typography>
```

Comment:
You can take both `Typography` components outside `Grid` since their layout will not vary with the size of the screen.